.github/workflows/unit_tests.yml

```diff
@@ -409,7 +413,8 @@ jobs:
 
       - name: Install build tools
         run: |
-          sudo apt update && sudo apt install -y clang-${{env.MULL_LLVM_VERSION}} ninja-build
+          sudo apt update && sudo apt install -y clang-${{env.MULL_LLVM_VERSION}} ninja-build python3-pip
+          sudo pip3 install --break-system-packages pytest pytest-forked hypothesis
```

Comment:
We want to do like https://github.com/intel/cicd-repo-infrastructure/blob/dev/.github/workflows/test.yml#L56